src/App.js

```diff
@@ -1,13 +1,61 @@
 import React from 'react';
+import TodoList from "./components/TodoComponents/TodoList"
+import TodoForm from "./components/TodoComponents/TodoForm"
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+state={
+  tasks: [
+    {
+      task: 'Organize Garage',
+      id: 1528817077286,
+      completed: false
+    },
+    {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+    }
+  ],
+  taskInput:""
+}
+
+changeHandler = e =>{
+  e.preventDefault();
+  this.setState({
+    // todos: this.state.todos.filter(task => !task.complete),
+    [e.target.name]: e.target.value
+  });
+};
+
+clearCompleted = e => {
+  e.preventDefault();
+
+  this.setState({
+    
+  })
+}
+
+addTask = e => {
+  e.preventDefault();
+  const newTask ={
+    task:this.state.taskInput,
+    id:Date.now(),
+    completed: false 
+  }
+  this.setState({
+    tasks: [...this.state.tasks, newTask],
+    taskInput: ''
+  })
```

Comment:
Great job both initiating a new task object and spreading it into a new array in the set state, as well as resetting the input after submission.